pkg/oc/cli/admin/policy/modify_roles.go

```diff
@@ -595,6 +595,18 @@ subjectCheck:
 	return newSubjects
 }
 
+func (o *RoleModificationOptions) checkRolebindingAutoupdate(roleBinding *roleBindingAbstraction) {
+	if roleBinding.Annotation(rbacv1.AutoUpdateAnnotationKey) == "true" {
+		if o.PrintErrf != nil {
+			o.PrintErrf("Warning: Your changes may get lost whenever a master"+
+				" is restarted, unless you prevent reconciliation of this"+
```

Comment:
Do we need to remind that updating cluster will also lost the changes.

Reply:
No, I upgrading a cluster requires masters restarts so it is implicit in this message already.